install/kubernetes/helm/istio/values.yaml

```diff
@@ -1,38 +1,38 @@
+# Use --set or additional values.yaml file to configure settings.
+# This file no longer uses sed, updateVersions.sh or istio.VERSIONS
+# TODO: evaluate if we need individual overrides for each component version, istio
+# is not typically tested with a mix of versions. Only supported case is version upgrade.
+
+# Common settings.
 global:
+  # Default tag for Istio images.
+  hub: istio
```

Comment:
is this hub correct?  Wouldn't it be docker.io/istio or gcr.io/istio?

Reply:
docker.io is added automatically, and it seems to work.

Reply:
wfm.

Reply:
would be good to specify it out as I'm not sure it'd work when custom registry is used in various cloud provider.  also for clarity :)

Reply:
Will do it in a separate PR, more changes will likely be needed before we change the tests.